Re-import images from WPT

nt1m opened the pull request:
#### 86897f61fc3f8a6e132345c32e290822c663b53c
<pre>
Re-import images from WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=298800">https://bugs.webkit.org/show_bug.cgi?id=298800</a>
<a href="https://rdar.apple.com/160497656">rdar://160497656</a>

Reviewed by Simon Fraser.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/5aa642def1c645781d2a26dd9a61209dc258d140">https://github.com/web-platform-tests/wpt/commit/5aa642def1c645781d2a26dd9a61209dc258d140</a>

* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/fill-and-stroke-styles/2d.pattern.image.broken-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-colorSpaceConversion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-invalid-args-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/offscreen/drawing-images-to-the-canvas/2d.drawImage.broken-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/offscreen/drawing-images-to-the-canvas/2d.drawImage.broken.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/images/100px-green-rect.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/images/README.md: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/anim-tao.png: Renamed from LayoutTests/imported/w3c/web-platform-tests/images/anim-poster-gr.png.
* LayoutTests/imported/w3c/web-platform-tests/images/anim-tao.png.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/animated-avif.avif: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/arrow-oriented-upright.jpg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/blue-png-cachable.py:
(main):
* LayoutTests/imported/w3c/web-platform-tests/images/broken.png:
* LayoutTests/imported/w3c/web-platform-tests/images/computer.jpg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/green-100x50.png:
* LayoutTests/imported/w3c/web-platform-tests/images/green.avif: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-100x50-alt.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-100x50.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-133x106.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-16x16.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-1x1.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-256x256.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-2x2.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-96x96.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/pattern-nosize.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/pattern.webm: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/undecodable.png: Copied from LayoutTests/imported/w3c/web-platform-tests/images/broken.png.
* LayoutTests/imported/w3c/web-platform-tests/images/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/images/webp-animated.webp: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/wide-gamut-pattern.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-darkblue-bg.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-lightblue-bg.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-orange-bg.svg:

Canonical link: <a href="https://commits.webkit.org/299934@main">https://commits.webkit.org/299934@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3df95eee6e0407f094a3eef6d675f16a633f195f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/120782 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/40476 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/31129 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/127187 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/72858 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/d80f9db0-5a7f-472f-9640-e587b3de8f88) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/41174 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/49053 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/91745 "Passed tests") | [![loading-orange](https://github-production-user-asset-6210df.s3.amazonaws.com/3098702/291015173-08c448be-ac0a-4fd6-92a3-8165057445b7.png) 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60989 "Build is in progress. Recent messages:Printed configuration") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/1d8210ce-5401-4513-9c6e-390f420d3c5e) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/123734 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32872 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/108266 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/72390 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/d45e2f56-b7f7-4b18-a049-0fbd4c6be648) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31902 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/26373 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/70783 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/102366 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/26551 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/130050 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/47703 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/36221 "Found 1 new test failure: http/tests/websocket/tests/hybi/contentextensions/block-cookies-worker.py (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/100362 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/48071 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/104444 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/100250 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/45642 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/23684 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/44372 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19167 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/47565 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/47034 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/50380 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/48720 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/3df95eee6e0407f094a3eef6d675f16a633f195f. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50683

nt1m committed:
Re-import images from WPT
https://bugs.webkit.org/show_bug.cgi?id=298800
rdar://160497656

Reviewed by Simon Fraser.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/5aa642def1c645781d2a26dd9a61209dc258d140

* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/fill-and-stroke-styles/2d.pattern.image.broken-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-colorSpaceConversion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-invalid-args-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/offscreen/drawing-images-to-the-canvas/2d.drawImage.broken-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/canvas/offscreen/drawing-images-to-the-canvas/2d.drawImage.broken.worker-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/images/100px-green-rect.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/images/README.md: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/anim-tao.png: Renamed from LayoutTests/imported/w3c/web-platform-tests/images/anim-poster-gr.png.
* LayoutTests/imported/w3c/web-platform-tests/images/anim-tao.png.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/animated-avif.avif: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/arrow-oriented-upright.jpg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/blue-png-cachable.py:
(main):
* LayoutTests/imported/w3c/web-platform-tests/images/broken.png:
* LayoutTests/imported/w3c/web-platform-tests/images/computer.jpg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/green-100x50.png:
* LayoutTests/imported/w3c/web-platform-tests/images/green.avif: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-100x50-alt.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-100x50.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-133x106.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-16x16.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-1x1.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-256x256.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-2x2.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/lcp-96x96.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/pattern-nosize.svg: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/pattern.webm: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/undecodable.png: Copied from LayoutTests/imported/w3c/web-platform-tests/images/broken.png.
* LayoutTests/imported/w3c/web-platform-tests/images/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/images/webp-animated.webp: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/wide-gamut-pattern.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-darkblue-bg.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-lightblue-bg.svg:
* LayoutTests/imported/w3c/web-platform-tests/images/wpt-logo/wpt-logo-orange-bg.svg:

Canonical link: https://commits.webkit.org/299934@main
